build.gradle.kts

```diff
@@ -62,20 +62,37 @@ dependencies {
   implementation("org.apache.commons:commons-text:1.9")
   implementation("com.pauldijou:jwt-core_2.11:5.0.0")
 
+  // https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk
+  implementation("com.amazonaws:aws-java-sdk-s3:$awssdkVersion")
+  implementation("com.amazonaws:aws-java-sdk-cloudformation:$awssdkVersion")
+  implementation("com.amazonaws:aws-java-sdk-core:$awssdkVersion")
+  implementation("com.amazonaws:aws-java-sdk-ec2:$awssdkVersion")
+  implementation("com.amazonaws:aws-java-sdk-kms:$awssdkVersion")
+  implementation("com.amazonaws:aws-java-sdk-sns:$awssdkVersion")
+  implementation("com.amazonaws:aws-java-sdk-sqs:$awssdkVersion")
+  implementation("com.amazonaws:jmespath-java:$awssdkVersion")
+
```

Comment:
CVE-2022 issue. had to exclusively state the version being used rather than from parent.